Tests/TuistGeneratorTests/Linter/StaticProductsGraphLinterTests.swift

```diff
@@ -1153,10 +1177,11 @@ class StaticProductsGraphLinterTests: XCTestCase {
             ]],
             dependencies: dependencies
         )
+        let config = Config.test()
         let graphTraverser = GraphTraverser(graph: graph)
 
         // When
-        let results = subject.lint(graphTraverser: graphTraverser)
+        let results = subject.lint(graphTraverser: graphTraverser, config: config)
 
         // Then
```

Comment:
is it worth adding one test cases which we know would have flagged a warning for one or more targets and add one of the exclusion list?

Reply:
I added a test for that scenario.